### ISSUE TITLE
Clicking with a mouse will make it impossible to change focus by code

### ISSUE THREAD

**Issue by bwmeyer** [dotnet/maui#10992]
### Description

When clicking with the mouse in android/android tv, changing the focus in a .net MAUI App by Element.Focus() will stop working.
To get it working again you need to change the focus manually by pressing any key on a keyboard.

I would assume that Element.Focus() should always force to keyboard mode.
Otherwise e.g. starting an app with a mouse on an android tv will make the use of a remote control completely unusable.
(The remote control has no tab key to change the focus to a button, reloading the page by clicking with the mouse will change the focus again...)

### Steps to Reproduce

1. Create a new MAUI Android App
2. Add a button "CloseButton"
3. Call CloseButton.Focus() in OnAppearing:
```
        protected async override void OnAppearing()
        {
            base.OnAppearing();
            Dispatcher.Dispatch(async () =>
               {
                   await Task.Delay(200);
                   CloseButton.Focus();
               });
        }
```
4. Start the android tv emulator
5. Click with the mouse somewhere in the android tv before starting the app
-> (no matter in app or launcher, just needs to change from keyboard to mouse input)
7. Start the App
> The "CloseButton" will not getting the keyboard focus.

### Link to public reproduction project repository

-

### Version with bug

6.0.486 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android TV 10.0, Android 8.0

### Did you find any workaround?

Start the app with the keyboard and don't use the mouse as long as the app should handle focus correctly.

**Comment by hartez** [comment]
Duplicate of #5983
